backend/src/channel-invited/channel-invited.service.spec.ts

```diff
@@ -120,7 +125,7 @@ describe('ChannelInvitedService', () => {
     })
 
     it('should find all ChannelInvited', async () => {
-      const resRequest = await channelInvitedService.findAllInChannel(
+      const resRequest = await channelInvitedService.findAllInChannelInvited(
```

Comment:
no need to change the name of this function, it's only for resolver, not services